.idea/misc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We should never upload .idea files; these are dev-environment specific. Please add ".idea" to the .gitignore file.

Reply:
Please also add .DS_Store to that file; those files should not be stored in git either.